carpooling/models/models.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
+
+class VehicleTrip(models.Model):
+    _name = "carpooling.vehicle_trip"
+    _description = "carpooling.vehicle_trip.description"
+
+    driver = fields.Many2one("res.users", "Driver", required=True, default=lambda self: self.env.user)
+    driver_uid = fields.Integer(compute="_get_driver_uid", store=True)
+
+    # field_lat = fields.Float(string="Latitude")
+    # field_lng = fields.Float(string="Longitude")
+
+    @api.depends('driver')
+    def _get_driver_uid(self):
+        for record in self:
+            record.driver_uid = record.driver.id
```

Comment:
We don't get the point ?  You can use `self.driver.id` everywhere it's needed and not use this field.
More over, store a record id in another field is quite redundant